tests/migrations/test_autodetector.py

```diff
@@ -1174,6 +1174,39 @@ def test_add_field_and_foo_together(self):
         self.assertOperationAttributes(changes, "otherapp", 0, 1, name="book", unique_together={("title", "newfield")})
         self.assertOperationAttributes(changes, "otherapp", 0, 2, name="book", index_together={("title", "newfield")})
 
+    def test_create_model_and_unique_together(self):
+        author = ModelState("otherapp", "Author", [
+            ("id", models.AutoField(primary_key=True)),
+            ("name", models.CharField(max_length=200)),
+        ])
+        book_with_author = ModelState("otherapp", "Book", [
+            ("id", models.AutoField(primary_key=True)),
+            ("author", models.ForeignKey("otherapp.Author", models.CASCADE)),
+            ("title", models.CharField(max_length=200)),
+        ], {
+            "index_together": {("title", "author")},
+            "unique_together": {("title", "author")},
+        })
+        before = self.make_project_state([self.book_with_no_author])
+        after = self.make_project_state([author, book_with_author])
+        autodetector = MigrationAutodetector(before, after)
+        changes = autodetector._detect_changes()
+        # Right number of migrations?
+        self.assertEqual(len(changes['otherapp']), 1)
+        # Right number of actions?
+        migration = changes['otherapp'][0]
+        self.assertEqual(len(migration.operations), 4)
+        # Right actions order?
+        action1 = migration.operations[0]
+        action2 = migration.operations[1]
+        action3 = migration.operations[2]
+        action4 = migration.operations[3]
+        self.assertEqual(action1.__class__.__name__, "CreateModel")
```

Comment:
I didn't change the order of AlterUniqueTogether and AlterIndexTogether, and I don't think that order is non-deterministic. I just wanted my test to pass even if that order changes for some reason - because it would still be the right order of actions. But I can modify it so it passes only for order the current code produces.